### DISCUSSION TITLE
Running test

### DISCUSSION THREAD

**Discussion by roy-augmentus** [tesseract-robotics/trajopt#284] [Q&A]
Hi, I want to run some of the test found in trajopt_sco,  any special thing do i have to do for that? Thanks

**Comment by Levi-Armstrong** [comment]
When building you can pass different compiler flags to build things like test, benchmarks and linters which can be found [here](https://github.com/tesseract-robotics/trajopt#building-trajopt-tests).

**Reply by roy-augmentus** [reply]
Thanks. Yes i have tried that and it works. 
Will it be possible to see the solver .lp file? I can't seem to find it anywhere

**Reply by roy-augmentus** [reply]
Nevermind. I found the location of the folder containing that file in my root folder. Thanks anyway!
